plutus-core/plutus-ir/src/PlutusIR/Compiler.hs

```diff
@@ -116,6 +118,11 @@ availablePasses =
     , Pass "case reduce"          (onOption coDoSimplifierCaseReduce)         (pure . CaseReduce.caseReduce)
     , Pass "known constructor"    (onOption coDoSimplifierKnownCon)           KnownCon.knownCon
     , Pass "beta"                 (onOption coDoSimplifierBeta)               (pure . Beta.beta)
+    , Pass "strictify bindings"   (onOption coDoStrictifyBindings)            (\t ->
```

Comment:
Can you rename `coDoStrictifyBindings` to `coDoSimplifierStrictify` or sth? The rest seem to have uniformly the `coDoSimplifier...` prefix